vyper/venom/ir_node_to_venom.py

```diff
@@ -270,6 +282,18 @@ def _returns_word(func_t) -> bool:
     return return_t is not None and _is_word_type(return_t)
 
 
+def _returns_stack_count(func_t) -> int:
+    ret_t = func_t.return_type
+    if ret_t is None:
+        return 0
+    if ENABLE_MULTI_RETURNS and isinstance(ret_t, TupleT):
```

Comment:
could probably generalize this to tuplelike (i.e. including StructT) but out of scope